docs/legacy_css.md

```diff
@@ -0,0 +1,115 @@
+# Whitehall Legacy CSS Styleguide
```

Comment:
It'd be useful to explain what legacy CSS is and when you might consider this guide. 

Are all the bits of this still relevant too? I wonder if there are bits that should be deleted or if even need this file at all.

Reply:
I think a lot of them might not be relevant, will look through and see what is not needed a bit more carefully. The other thought I had was to just delete this file as we're no longer planning to support it.

Reply:
Sure, I think it's likely fine to delete it only seems useful if we expect people to read it to know how to do something.

Reply:
I have added a notice for users to make sure they understand it's legacy and will be deleted once bootstrap can be removed.